### PR TITLE
Merge default

### LINKED ISSUE #920: odd behavior with spreadsheet-for-component for several years
http://rendering.secxbrl.info/#?url=https:%2F%2Fsecxbrl.28.io%2Fv1%2F_queries%2Fpublic%2Fapi%2Fspreadsheet-for-component.jq%3Ftoken%3Dc3049752-4d35-43da-82a2-f89f1b06f7a4%26format%3Djson%26%26cik%3D0000804328%26fiscalYear%3D2013%26fiscalPeriod%3DFY%26fiscalPeriodType%3DYTD%26fiscalPeriodType%3Dinstant%26disclosure%3DIncomeStatement%26override%3Dtrue

image

The value for "Licensing" for 2013 disappears if the year 2014 is added:

http://rendering.secxbrl.info/#?url=https:%2F%2Fsecxbrl.28.io%2Fv1%2F_queries%2Fpublic%2Fapi%2Fspreadsheet-for-component.jq%3Ftoken%3Dc3049752-4d35-43da-82a2-f89f1b06f7a4%26format%3Djson%26%26cik%3D0000804328%26fiscalYear%3D2013%26fiscalYear%3D2014%26fiscalPeriod%3DFY%26fiscalPeriodType%3DYTD%26fiscalPeriodType%3Dinstant%26disclosure%3DIncomeStatement%26override%3Dtrue

image

### PR DESCRIPTION
**Pull request by gislenius**
Fixes #920

### CODE REVIEW

File: queries/public/test/edinet/spreadsheet-for-component-expected-bs-one-en.jq [hunk 1]
```diff
@@ -6,8 +6,8 @@
       "Label" : "四半期連結貸借対照表"
     }, 
     "Hypercube" : {
-      "Name" : "xbrl:UserDefinedHypercube", 
-      "Label" : "User-defined Hypercube"
+      "Name" : "jppfs-cor:BalanceSheetTable", 
+      "Label" : "貸借対照表"
```
Comment (dknochen) [inline queries/public/test/edinet/spreadsheet-for-component-expected-bs-one-en.jq]: the changes in the expected test results look unrelated. Can you explain why this is needed? Especially, because the file ends with "en", but the result changed to japanese.

**Comment by dknochen** [comment]
Optionally, we could also add this call to the tests and check that it has 2 times Licensing in it (checking for the 2 different concepts: qcom:LicensingAndRoyaltyFees and us-gaap:LicensesRevenue). Feel free to skip if you think it is overdoing it.

**Comment by gislenius** [comment]
@dknochen which call? :-)

**Comment by gislenius** [comment]
Hi Dennis,

You're right about the unrelated changes. That was a bug with default languages when merging components. It's solved in #946. I will adapt this PR when #946 gets merged.

Kind regards,
Ghislain

**Comment by dknochen** [comment]
> which call?

I meant the call from the bug report: https://github.com/28msec/cellstore/issues/920

**Comment by dknochen** [comment]
#946 was merged, but the changes are still there. Could you take a look? 

thank you :)

**Comment by gislenius** [comment]
Hi Dennis,

The changes (en/ja) are no longer visible. The ones visible in the test expected results are normal: in the one* tests, no merge happens because there's a single component. The rest of the diff (change of order of the fields) is irrelevant and also due to the non-merge.

Does it make sense?

Kind regards,
Ghislain

**Comment by gislenius** [comment]
@dknochen I added the test. Back to you!

**Comment by dknochen** [comment]
@gislenius ok , I get it now. This change:

image

is due to the fact, that there is no merge done anymore, but was done before. And, merge had renamed the hypercube to "User defined hypercube". 

thank you.
